test/utility.test.ts

```diff
@@ -48,7 +48,20 @@ suite("utility tests", () => {
     assert.equal(moduleToImageMap.size, 4);
     assert.equal(imageToDockerfileMap.size, 4);
     assert.equal(imageToBuildOptions.size, 4);
-    assert.equal(imageToBuildOptions.get("localhost:5000/samplemodule:0.0.1-amd64").length, 7);
+    assert.equal(imageToBuildOptions.get("localhost:5000/samplemodule:0.0.1-amd64").length, 8);
+
+    const filteredOption = imageToBuildOptions.get("localhost:5000/samplemodule:0.0.1-amd64").filter((value, index) => {
```

Comment:
We'd better to wrap the filter logic into a method, so that the test code and prod code could share.

Reply:
The current logic is wrapped in the private method constructBuildCmd.  I am thinking of add anther utility public method, but seems a little bit over-wrapped. Another engineer task could in our backlog is to wrap the "module.json" schema into a class. So all the stuff could be checked in the class. I will raise an issue in github for tracking.